layer/tests/layer_tests_main.h

```diff
@@ -15,15 +15,15 @@
 
 #pragma once
 
-#ifdef ANDROID
+#ifdef VK_USE_PLATFORM_ANDROID_KHR
```

Comment:
I'm not understanding why `layer_tests_main.cpp` uses `#ifdef __ANDROID__` and this code uses `#ifdef VK_USE_PLATFORM_ANDROID_KHR`.  Comments would be welcome.

Reply:
You are right, I should use __ANDROID__. I am not use VK_USE_PLATFORM_ANDROID_KHR is really useful.

I'll fix this in next PR.